packages/squiggle-lang/src/public/SqPlot.ts

```diff
@@ -0,0 +1,32 @@
+import { LabeledDistribution, Plot, vPlot } from "../value";
+import { wrapDistribution } from "./SqDistribution";
+import { SqPlotValue } from "./SqValue";
+import { SqValueLocation } from "./SqValueLocation";
+
+export class SqPlot {
+  constructor(private _value: Plot, public location: SqValueLocation) {}
+
+  get distributions() {
+    return this._value.distributions.map(({ name, distribution, opacity }) => ({
+      name,
+      distribution: wrapDistribution(distribution),
+      opacity,
+    }));
+  }
+
+  get showLegend() {
+    return this._value.showLegend;
+  }
+
+  get colorScheme() {
```

Comment:
There are questions here as to whether this needs to be exposed? I could probably argue that it doesn't...